README.md

```diff
@@ -45,6 +45,16 @@ the 5th version built against Kubernetes 1.20 APIs.
 This controller is developed and tested against Rackspace OpenStack Public Cloud using Go 1.16.
 Pull requests are welcome.
 
+## How to run locally. 
+
+1. In order to test your changes you will need a cloudspace. If you don't have a cloudspace then create one.
```

Comment:
Change all references of cloudspace to kubernetes cluster. 
Cloudspace is a spot only concept and doesn't make sense to use that term in a generic component like the CCM here.